components/ModalDelete.js

```diff
@@ -0,0 +1,32 @@
+import styled from "styled-components";
+
+export default function ModalDelete({ onConfirm, onCancel }) {
+  return (
+    <StyledBackground>
+      <StyledContainer>
+        <p>Are you sure you want to delete this project?</p>
+        <button onClick={onConfirm}>Yes</button>
+        <button onClick={onCancel}>No</button>
+      </StyledContainer>
+    </StyledBackground>
+  );
+}
+
+const StyledBackground = styled.div`
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.5);
+`;
+
+const StyledContainer = styled.div`
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  background-color: white;
+  border: 1px solid black;
+  padding: 0px 20px 20px 20px;
+`;
```

Comment:
Not 100% sure but you might not need `position: fixed` here since the background is already fixed and centers this container - please check.

Reply:
Without `position: fixed` the modal ends up in the upper left corner.